src/engine/client/cl_console.cpp

```diff
@@ -1270,10 +1270,13 @@ void Con_ScrollToTop()
 	//consoleState.bottomDisplayedLine = consoleState.scrollLineIndex;
 }
 
-void Con_ScrollToBottom()
+void Con_ScrollToBottom(bool instantly)
 {
 	consoleState.scrollLineIndex = consoleState.lines.size() - 1;
-	//consoleState.bottomDisplayedLine = consoleState.currentLine;
+	if (instantly)
```

Comment:
A crash occurred because `bottomDisplayedLine` is used [here](https://github.com/DaemonEngine/Daemon/blob/master/src/engine/client/cl_console.cpp#L885) to index into the lines array.